src/openforms/submissions/api/serializers.py

```diff
@@ -267,12 +268,17 @@ class Meta:
     @elasticapm.capture_span(span_type="app.api.serialization")
     def to_representation(self, instance):
         # invoke the configured form logic to dynamically update the Formio.js configuration
-        new_configuration = evaluate_form_logic(
-            instance.submission,
-            instance,
-            instance.submission.data,
-            **self.context,
-        )
+        try:
+            new_configuration = evaluate_form_logic(
+                instance.submission,
+                instance,
+                instance.submission.data,
+                **self.context,
+            )
+        except Exception as e:
+            if detail := getattr(e, "detail", None):
+                raise APIException(detail)
+            raise e
```

Comment:
This is in preparation to display a proper error message to the user in case the referentielijsten options can't be fetched. I'm not sure if this is the right approach to get a specific message. It currently looks like this:

![image](https://github.com/user-attachments/assets/fd8c1232-bf49-481f-924b-9f855e53d63c)

Preferably I think the "detail" message should be displayed in the error block, which would require changes in the SDK